java/sim-pose-estimation/src/main/java/frc/sim/DrivetrainSim.java

```diff
@@ -0,0 +1,78 @@
+package frc.sim;
+
+import org.photonvision.SimVisionSystem;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.simulation.AnalogGyroSim;
+import edu.wpi.first.wpilibj.simulation.DifferentialDrivetrainSim;
+import edu.wpi.first.wpilibj.simulation.EncoderSim;
+import edu.wpi.first.wpilibj.system.LinearSystem;
+import edu.wpi.first.wpilibj.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.system.plant.LinearSystemId;
+import edu.wpi.first.wpiutil.math.numbers.N2;
+import frc.robot.Constants;
+
+
+public class DrivetrainSim {

```

Comment:
Not sure this class is ever used

Reply:
Derp should have marked the commit as WIP. DrivetrainSim still needs to be integrated into robot.java - it should get used soon.

Architecturally - Goal will be to have a clean split - all code in the `sim` folder won't be called at all on a real robot.